### DISCUSSION TITLE
How can I create a Beamer presentation with Quarto from jupyter notebook?

### DISCUSSION THREAD

**Discussion by Mgonrub** [quarto-dev/quarto-cli#5842] [Q&A] [answered]
### Description

I'm trying to replicate the output from the following address:
[https://github.com/quarto-dev/quarto-gallery/tree/main/presentations/beamer](url)

I have done the following:

1. Install Quarto.
2. Download an example beamer.qmd
3. Change it to a jupyter notebook file to view with: `quarto convert beamer.qmd`
4. To output it as a pdf file I used: fourth render `beamer.ipynb --to pdf`. But first I had to install the following `quarto install tinytex`.

I'm using Windows 10 and Python 3.9.0 and jupyter notebook and installed Quarto a couple of days ago, but I can't figure out how to render an ipynb document as a beamer presentation. And also I would like to use a theme from what is available "Beamer theme gallery". Maybe my query is very basic, but if someone can help me I would be very grateful. Please.

Could you please help me with the steps and tools needed to render a file.ipynb to a beamer-type presentation using a format, for example, from the beamer gallery.

jupyter notebook --version
6.4.6
quarto --version
1.3.353

**Comment by dragonstyle** [comment, marked as answer]
Hello and welcome!

You should render the document to the `beamer` format, like:

`quarto render beamer.ipynb --to beamer`

That should do it!

You can read more about Beamer specific options, including themes, here:

https://quarto.org/docs/presentations/beamer.html#beamer-options

**Comment by Mgonrub** [comment]
My most humble thanks for your immense help.
Your advice was effective and the result is as expected. Now to play with the options.

Marco
